test/fixtures/testCases/serde.js

```diff
@@ -0,0 +1,141 @@
+'use strict';
+
+module.exports = [
+  {
+    name: 'true',
+    value: true,
+    serialized: 'true'
+  },
+  {
+    name: 'false',
+    value: false,
+    serialized: 'false'
+  },
+  {
+    name: 'integer',
+    value: 42,
+    serialized: '42'
+  },
+  {
+    name: 'negative integer',
+    value: -3,
+    serialized: '-3'
+  },
+  {
+    name: 'float',
+    value: 1e100,
+    serialized: '1e+100'
+  },
+  {
+    name: 'float with comma',
+    value: 3.14,
+    serialized: '3.14'
+  },
+  {
+    name: 'smallFloat',
```

Comment:
Style of this name is inconsistent with overall test naming style in this tests, it should be `'small float'`.